engine/src/test/java/org/camunda/bpm/engine/test/history/HistoricTaskInstanceTest.java

```diff
@@ -64,6 +65,7 @@ public void testHistoricTaskInstance() throws Exception {
 
     // Set priority to non-default value
     Task runtimeTask = taskService.createTaskQuery().processInstanceId(processInstanceId).singleResult();
+    assertEquals("Created", runtimeTask.getTaskState());
```

Comment:
:x: The focus of this test class is historic task instances. I suggest moving out the assertion for runtime task to another test class, for example `TaskServiceTest`. 
(Another argument to move out this assertions: the test here runs with history level set to audit and full so we don't test the changes where the history level is not configured.)
```suggestion
```